src/test/java/org/simulator/oven/SCPSolverIssue.java

```diff
@@ -17,11 +18,11 @@ public static void main(String[] args) {
 
     LinearProgramSolver solver = SolverFactory.newDefault();
     double[] sol = solver.solve(lp);
-    System.out.println(sol);
```

Comment:
ErrorProne complained about this: printing an array directly does not show anything useful.